src/components/create-todo/index.js

```diff
@@ -1,12 +1,13 @@
 import { useState } from 'react';
+import './create-todo.css';
 
 export default function CreateTodo({ addTodo }) {
```

Comment:
At line 38, I think you can use `input` tag instead of `textarea` and make sure to set `type` attribute to `text` because `textarea` tag is useful in cases where the user has to input large amount of text, and in this case `input` tag is a better option!